src/Shared/TaskLoggingHelper.cs

```diff
@@ -739,14 +774,14 @@ public void LogErrorWithCodeFromResources(string messageResourceName, params obj
         }
 
         /// <summary>
-        /// Logs an error using the specified resource string and other error details. 
+        /// Logs an error using the specified resource string and other error details.
         /// If the message has an error code prefixed, the code is extracted and logged with the message. If a
         /// help keyword prefix has been provided, a help keyword for the host IDE is also logged with the message. The help
         /// keyword is composed by appending the error message resource string name to the prefix.
-        /// 
+        ///
         /// A task can provide a help keyword prefix either via the Task (or TaskMarshalByRef) base class constructor, or the
         /// Task.HelpKeywordPrefix (or AppDomainIsolatedTask.HelpKeywordPrefix) property.
-        ///    
```

Comment:
Would it be easy for you to revert these unrelated whitespace fixes?

Reply:
Looks like my editor (VS Code) is actively removing these. Not entirely sure how to fix that to be honest, but I'll check.

Reply:
Since I was on Windows with Visual Studio I managed to fix these. Let me know if this is okay now.